ntfs2ntfs/src/main.rs

```diff
@@ -41,6 +41,19 @@ struct Opt {
         default_value = &transit_model::CURRENT_DATETIME
     )]
     current_datetime: DateTime<FixedOffset>,
+
+    // The max distance in meters to compute the tranfer
+    #[structopt(long, short = "d", default_value = "500")]
```

Comment:
This letter being taken for 2 other processes on the TT side we may change these latter processes (to harmonize, and keep "d", "s", "t") :wink:

Reply:
processes that need transfers? I'm not sure

Reply:
Effectively. I just looked at those who had ntfs out :+1: